pkgs/development/tools/build-managers/cmake/default.nix

```diff
@@ -61,13 +61,15 @@ stdenv.mkDerivation rec {
   outputs = [ "out" ] ++ lib.optionals buildDocs [ "man" "info" ];
   setOutputFlags = false;
 
-  setupHook = ./setup-hook.sh;
+  setupHooks = [
+    ./setup-hook.sh
+    ./check-pc-files.sh
```

Comment:
Can we put everything in the main setup hook? It's much easier to browse all the stdenv magic in one predictable place.

Reply:
we can but i want to keep them separate

I've renamed the hook to `check-pc-files-hook.sh`

Reply:
Why?

Reply:
because it is a hook and hooks mostly end in ``-hook.sh``

Reply:
That's not the question. Why @Artturin wants a separate file?

Reply:
Its a separate hook and there's no reason to keep them in the same file

all hooks can be found with `fd --type f "hook.sh"`